vnext/Desktop/WinHTTPWebSocketResource.cpp

```diff
@@ -0,0 +1,447 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// clang-format off
+#include "pch.h"
+
+#include <Unicode.h>
+#include <Utils.h>
+#include "WinHTTPWebSocketResource.h"
+
+// Windows API
+#include <comutil.h>
+
+using Microsoft::Common::Unicode::Utf8ToUtf16;
+
+using std::function;
+using std::size_t;
+using std::string;
+using std::wstring;
+
+namespace Microsoft::React
+{
+
+WinHTTPWebSocketResource::WinHTTPWebSocketResource(const string& urlString)
+  : m_openSessionFlags{ WINHTTP_FLAG_ASYNC }
```

Comment:
>m_openSessionFlags [](start = 4, length = 18)

please make these assignments in .h, here we should assign values coming from constructor's args